MAUI/DataGrid/grid-events.md

```diff
@@ -59,6 +73,19 @@ private void dataGrid_CellDoubleTapped(object sender, DataGridCellDoubleTappedEv
 {% endhighlight %}
 {% endtabs %}
 
+### CellDoubleTappedCommand and its Parameter
+When we double tapped the cell, the `CellDoubleTappedCommand` triggered that binds to the command which you have defined in your ViewModel. With this we can also use the `CellDoubleTappedCommandParameter` to be passed to the `Command` when it is executed.
```

Comment:
The command will be invoked while double tapping a cell in the SfDataGrid. The instance of `DataGridCellDoubleTappedEventArgs` will be passed as the default value of the command's parameter. It can be customized by setting the `CellDoubleTappedCommandParameter` property.

Reply:
Changed